docs/core-concepts/concepts.md

```diff
@@ -10,9 +10,9 @@ Karmada uses the Kubernetes Native API definition for the federated resource tem
 
 ## Propagation Policy
 
-Karmada offers a standalone Propagation(placement) Policy API to define multi-cluster scheduling and spreading requirements.
+Karmada offers a standalone Propagation (placement) Policy API to define multi-cluster scheduling and spreading requirements.
 
-- Support 1:n mapping of policy: workload. Users don't need to indicate scheduling constraints every time creating federated applications.
+- Support 1:n mapping of policy:workload. Users don't need to indicate scheduling constraints every time creating federated applications.
```

Comment:
```suggestion
- Support 1:n mapping of policy:workloads. Users don't need to indicate scheduling constraints every time creating federated applications.
```
How about appending the suffix `s`, so `policy:workloads` has a consistent pattern with `1:n`?

Reply:
+1 nice finding